README.md

```diff
@@ -21,6 +23,37 @@ Many thanks for that outstanding original theme, Radek!
 For more information about this fork and the differences to the original theme, please see:
 [Changes compared to the original theme](#changes-compared-to-the-original-theme) below.
 
+<!-- omit from toc -->
+## Table of contents
```

Comment:
Adding this TOC would only make sense if the README had some tooling to re-add it. I don't know which tool was used.

Also given that GitHub now has a dropdown list of headings, I'm reluctant to keep it, as I don't see much of a need. Please remove, along with the `<!-- omit from toc -->` comments.

Reply:
I used the `yzhang.markdown-all-in-one` VSCode extension.